src/python_testing/TC_MCORE_FS_1_3.py

```diff
@@ -36,14 +36,10 @@ class TC_MCORE_FS_1_3(MatterBaseTest):
     @async_test_body
```

Comment:
Blocking comment that should only be resolved after test plan spec PR is merged first
https://github.com/CHIP-Specifications/chip-test-plans/pull/4575